Jersey Choice (#104)
RoboCup-SPL/Rules: rules/robot_players.tex

\end{itemize}

The jersey sets that are used in a game are decided by the head referee in accordance with the teams during the pre-game meeting (\cf~\cref{sec:referee_team_communication}), subject to the following conditions:
\begin{itemize}
  \item Both teams' \emph{field player} colors must be easily distinguishable from each other.
  \item Both teams' \emph{goalkeeper} colors must be easily distinguishable from both teams' \emph{field player} colors.
  \item The two \emph{goalkeeper} colors may only be similar when this is the case for all choices among the available jersey sets.
\end{itemize}

> The head referee should respect the teams' wishes unless they violate the above conditions, in which case the head referee must bring about a solution.

Reviewer: ```suggestion
The head referee should respect the teams' wishes unless they violate the above conditions, in this case the head referee comes up with a solution.
```
Sounded a little bit weard for me. That is the only point here.

Author: I'm not 100% sure, but I think that "in which case" can join two clauses and "in this case" cannot. If you wanted "in this case", you'd have to split it into two sentences (or perhaps use a semicolon, but to me, a comma doesn't "feel" enough to separate the clauses). "in which case" is definitely a valid expression here.

For the second part, I agree that "bring about" is an uncommon phrase, as I didn't know it before I was looking for a translation of the German word "herbeiführen" (as in "der Schiedsrichter muss eine Lösung herbeiführen").
